embed/config.go

```diff
@@ -683,7 +684,7 @@ func (cfg *Config) GetDNSClusterNames() ([]string, error) {
 		zap.Error(httpCerr),
 	)
 
-	return clusterStrs, cerr
+	return clusterStrs, multierr.Combine(cerr, httpCerr)
```

Comment:
Could you, please, add comments to "GetDNSClusterNames" signature that the method might return combination of the experienced errors and actual collected results ?

Reply:
Done